### DISCUSSION TITLE
How to handle outliers in lab metrics

### DISCUSSION THREAD

**Discussion by guanchengcheng** [MIT-LCP/mimic-code#1641] [MIMIC-IV]
When I performed the first day of laboratory index extraction, there were many outliers (tens of thousands) in the triglyceride results I extracted, how to define whether the triglyceride result is NULL like the laboratory index on the first day.

I hope my questions are clear. Thank you in advance.

**Comment by alistairewj** [comment]
Really depends on how you define outlier and why you are extracting your data. Hard to give any advice without understanding more about your problem.

**Reply by guanchengcheng** [reply]
Thank you very much for your help.
